index.html

```diff
@@ -4,10 +4,217 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="mobile-version-style.css">
+    <title>My Portfolio Mobile Version</title>
+    <link rel="shortcut icon" href="Images/favicon.ico"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+   <header class="presentation">
+      <nav class="nav-container">
+         <img class="my-logo" src="Images/Logo HG.png">
+         <i style="color: #acb7c3" class="fa fa-bars"></i>
+      </nav>
```

Comment:
- [Optional] Kindly consider removing the margins at the top of the header so that the page will look like the template given.
### Your design
![image](https://user-images.githubusercontent.com/94551484/221296256-21d9f883-3b95-4be8-9b66-b02c729c004b.png)

### Figma 
![image](https://user-images.githubusercontent.com/94551484/221296363-9e479152-d995-4797-b495-0c229b4bccd0.png)